federation/pkg/typeadapters/secret.go

```diff
@@ -27,20 +27,22 @@ import (
 	kubeclientset "k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
 )
 
+const secretKind = "secret"
```

Comment:
better to use camel case for secretKind, it is the convention used for kube object kind.